File: Codigo_Fonte_pascal/lista3/lista3_2.pas
program media_vetor;

var
   lista_nota : array[1..10] of real;
   i : integer;
   media, soma_nota : real;

begin
     for i := 1 to 10 do
         readln(lista_nota[i]);
     for i:= 1 to 10 do
         soma_nota := soma_nota + lista_nota[i];
     media := soma_nota / 10;
     writeln('A media aritmetica : ', media);
     readln;
end.

File: Codigo_Fonte_c/Lista 3/lista3_2.c
#include <stdio.h>

int main(){
	int i;
	float lista_nota[10];
	float media, soma_nota = 0;
	for(i = 0; i < 10; i++){
		scanf("%f", &lista_nota[i]);
	}
	for (i = 0; i < 10; i++){
		soma_nota += lista_nota[i];
	}
	media = soma_nota / 10;
	printf("A media aritmetica : %f", media);

	return 0;	
}
